### ISSUE TITLE
kdtree does not install cleanly from PyPI

### ISSUE THREAD

**Issue by ntamas** [stefankoegl/kdtree#30]
In a clean Python virtualenv, `kdtree` does not install cleanly with `pip install kdtree` because `bounded_priority_queue.py` is missing from the uploaded package:

```
$ pip install kdtree
Collecting kdtree
  Downloading kdtree-0.14-py2.py3-none-any.whl
Installing collected packages: kdtree
Successfully installed kdtree-0.14

$ python
Python 2.7.10 (default, Sep  4 2015, 19:22:34)
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.51)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import kdtree
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/site-packages/kdtree.py", line 16, in <module>
    from bounded_priority_queue import BoundedPriorityQueue
ImportError: No module named bounded_priority_queue
```

**Comment by andreasellervee** [comment]
Yup, I had the same problem. The temporary workaround is copy `bounded_priority_queue.py` from this repo and add to your path.

**Comment by stefankoegl** [comment]
I've just released 0.15 which should fix this problem.
